app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    mv, source_file, path_file = command.split()
+    if mv == "mv" and len(command.split()) == 3:
+        if "/" not in path_file:
+            os.rename(source_file, path_file)
+        elif "/" in path_file:
+            full_path_file = path_file
+            path_file = path_file.split("/")
+            new_file_name = path_file[-1]
+            path_file.pop(-1)
+            path_file = "/".join(path_file)
+            if not os.path.exists(path_file):
+                os.makedirs(path_file)
+            if os.path.exists(new_file_name):
+                os.remove(new_file_name)
```

Comment:
Redundant, because in the write mode you will anyway rewrite data from this file